bin/_helm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eu
+
+# Checks if target/helm exists and it installs it if it doesn't.
+# Use `install_helm true` to also install Tiller.
+install_helm() {
+    bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+    rootdir="$( cd $bindir/.. && pwd )"
+
+    if [ ! -f $rootdir/target/helm ]; then
```

Comment:
name the executable with the version number, similar to protoc, that way when we update the version in this script, it'll re-download.